rst2man.py

```diff
@@ -1081,6 +1080,7 @@ def depart_warning(self, node):
         self.depart_admonition()
 
     def unimplemented_visit(self, node):
+        raise NotImplementedError(node.astext())
```

Comment:
Is this needed? There's already an exception raised on the next line...

Reply:
My mistake, this line can be removed.